### PR TITLE
Module 1: Details popup

### PR DESCRIPTION
**Pull request by Shedrack-Sunday**
## Changes made include:

1. A user click (or tap) button to check project details, the popup with details about the project appears.
>When the user clicks (or taps) the close (X) button, the popup disappears.

2. The JavaScript array is used to store all the information for all projects.
>the following pieces of data were stored in a JavaScript object: name, description, featured image, technologies, link to live version, and link to the source.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -721,3 +720,170 @@ li {
...
+  color: white;
+  padding: 12px 12px;
+  text-align: center;
+  font-size: 17px;
+  border: none;
+  height: 48px;
+  gap: 12px;
+}
+
+@media (min-width: 1050px) {
```
Comment (MahmoudBakr23) [inline style.css]: - [x] Well done styling the popup window so far. However, you may have to write some more CSS code to shape the widths and heights of it across multiple screen sizes. Please take a look at the two images down below.
- [x] Please fix the styling of the `X` button
**Desktop**
Screenshot (2)
Screenshot (1)
**Mobile**
Screenshot (3)

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Shedrack-Sunday 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done with following the Figma design
- Great work with javascript
- Descriptive PR and Readme file
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process improves the code you wrote, not criticizes it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

- Please fix the MIT link for a professional README file.

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention me @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -721,3 +720,192 @@ li {
...
+    letter-spacing: 0.03em;
+    color: #344563;
+    border-radius: 0.5em;
+  }
+
+  .card-work-details-mobile {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 60px;
+    grid-template-rows: 60px repeat(5, auto);
+  }
```
Comment (leonardodiasb) [inline style.css]: - [ ] As the previous Code Reviewer mentioned, you should apply some style to your mobile to make it look like the Figma template and fit the screen of the user. There are some differences of your style from the Figma template like the spacing between the buttons, the fonts inside the button, the colors of the button, the margin around the modal, etc. Also on different mobile screens like the Iphone SE, the modal does not fit inside the screen making it possible to scroll down, which is not a good user experience. Please fix the responsiveness of your modal to make it fit on every screen size and also to look more like the Figma template.
 - [ ] The same considerations above applies to the desktop view. It looks a little bit different from the Figma template. The design is not following the Figma template desktop design, it is following the same mobile design, and its larger than the viewport. Please try to make your modal look as much as possible as the Figma template.
 -Iphone SE view:
image
-Desktop view:
image
-Figma template desktop view:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Shedrack-Sunday, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

Great work with javascript ✔️ 
Descriptive PR and Readme file ✔️ 
Correct Linters ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Shedrack, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the CR's instructions.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Shedrack-Sunday** [comment]
> Hi Shedrack,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> ## Highlights
> * good job following the CR's instructions.
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hello Micheal, please check your approval for this project as it's not showing on my dashboard. Thanks @mikethreels

**Pull request merged**